.huskyrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  hooks: {
+    'pre-commit': 'lint-staged',
+  },
+};
```

Comment:
is it really necessary to have it separate file instead of including it in package.json?

Reply:
We removed prettierrc and added into package.json.. why not doing the same with this one?

Reply:
Probably not.